### DISCUSSION TITLE
Should PocketBase allow admins to login with an username?

### DISCUSSION THREAD

**Discussion by qgustavor** [pocketbase/pocketbase#2873] [Q&A]
Currently users can login with just a username without requiring a email address to be set, but that's not available to admins. Can't this feature be enabled for admins too? It's not like forgetting an admin password is a big issue now since the `pocketbase admin` command exists.

**Comment by ganigeorgiev** [comment]
There are no plans for now to add other admin authentication methods (_with exception maybe to webauthn, but that will be far in the future_).

The email domain part doesn't really need to be real (_the MX validator was disabled in https://github.com/pocketbase/pocketbase/discussions/152#discussioncomment-3534061_) and you can treat it as a "username".

**Reply by qgustavor** [reply]
I saw that comment when researching if someone already suggested it before. It's not really disabled in a way it can be used like an username:

![image](https://github.com/pocketbase/pocketbase/assets/1020670/1cdfad87-4540-4406-8f3f-516142e443e5)

The web interface still expects an email address.

![image](https://github.com/pocketbase/pocketbase/assets/1020670/1d788213-b63b-473f-9d51-c80db6633f59)

It still disallows some valid email addresses. Notice that, this time, Chromium email validation passed, so it didn't display an error balloon, but PocketBase's validation failed.

One possibility is allowing `localhost` as a valid TLD for admin emails, but I would still prefer an username over an email.
